### ISSUE TITLE
Prefix `:read-only` psuedo class

### ISSUE THREAD

**Issue by jacobbuck** [thysultan/stylis.js#35]
Input:

```css
/* @scope .foo */

input:read-only {
	color: red;
}
```

Expected result:

```css
.foo input:-moz-read-only {
	color: red;
}

.foo input:read-only {
	color: red;
}
```

Actual result:

```css
.foo input:read-only {
	color: red;
}
```

**Comment by thysultan** [comment]
should land in the next patch version.

**Comment by thysultan** [comment]
landed with 3.1.6
